### ISSUE TITLE
Backslashes incorrectly interpreted in Markdown verbatim text in HTML output

### ISSUE THREAD

**Issue by TransientResponse** [copperspice/doxypress#21]
When a DoxyPress project has a Markdown main page with verbatim text (e.g. a source code listing), backslash-escaped text is incorrectly interpreted in the resulting HTML view.

For example, the markdown
```
~~~
cout << "A thing" << "\n";
~~~
```

Produces the HTML code listing
```
cout << "A thing" << "n";
```

Ergo the backslash is removed. This should not happen as this is verbatim text and should be transcribed verbatim. 

Note that LaTeX output is not affected by this, and the backslash remains in the generated LaTeX. 

I will update this issue with more information when I have time to test it more thoroughly.

**Comment by TransientResponse** [comment]
I just confirmed that this issue also affects other Markdown pages as well, not just the main page. Seems to be a general issue with verbatim text in Markdown.

**Comment by agserm** [comment]
This turned out to be an issue with all code blocks, unrelated to markdown. Fixed in bf95197, thanks for your report!
